rollup.config.js

```diff
@@ -20,14 +21,27 @@ function checkInternals() {
   return fs.existsSync('./node_modules/@qlik');
 }
 
+const pkg = require(path.resolve(process.cwd(), 'package.json')); // eslint-disable-line  import/no-dynamic-require
+const { name, version, license, author } = pkg;
+
+const auth = typeof author === 'object' ? `${author.name} <${author.email}>` : author || '';
+const moduleName = name.split('/').reverse()[0];
+
+const banner = `/*
+* ${name} v${version}
+* Copyright (c) ${new Date().getFullYear()} ${auth}
+* Released under the ${license} license.
+*/
+`;
```

Comment:
What does this do? Is it added to the DOM? Is it visible?

Reply:
nah, it just sits at the top of the dist file

Reply:
ah ok